### ISSUE TITLE
Issue with --concat-audio

### ISSUE THREAD

**Issue by Wea1thRS** [leoncvlt/blinkist-scraper#10]
A lot of blinks download the audio, then when it comes to combining, it must error but all the audio files get deleted.

**Comment by leoncvlt** [comment]
Does it display any error in the terminal? Do you have ffmpeg installed and added to the PATH?

**Comment by Wea1thRS** [comment]
No errors, no. Yes I have ffmpeg, --concat-audio works for many of them, just these specific ones work, then get deleted. I run the program again and it skips the ones I have audios for, but, when it gets to the ones that it deleted, it re-gets and then deletes again.

**Comment by Wea1thRS** [comment]
Video of how it behaves:
https://streamable.com/w8cry5

Screenshot of the terminal output:
https://i.imgur.com/JnHhWen.png

**Comment by rocketinventor** [comment]
Hi @Wea1thRS,

I haven't had issues like what you described in production, but I have seen something like it while messing around with the concat audio code.

The issue that I found was that the paths to the audio files were incorrect, meaning FFMPEG did not know where to find the files.

To help debug this, we need to see the `temp.txt` file that is generated with the paths for the audio...

Instructions:
Go to the file: `generator.py` and find the lines (near the bottom):
```python
  if (os.path.exists(files_list)):
    os.remove(files_list)
```
Then delete/comment out the line:
```python
os.remove(files_list)
```

My recommendation is to replace it with a print line, such as:
```python
  #os.remove(files_list)
  print(f"os.remove({files_list})") # for debugging
```

...And then find that text file and copy it here (or check yourself if the paths are correct).

**Comment by Wea1thRS** [comment]
Upon looking, it creates the concat'd file just fine, however, it fails to tag for reasons that I am unsure. Maybe the title is too long? I can't seem to figure this out.

**Comment by rocketinventor** [comment]
It seems similar to [Issue #4](https://github.com/leoncvlt/blinkist-scraper/issues/4#issuecomment-611517519).

Did you follow the instructions in the readme for long filenames? (https://github.com/leoncvlt/blinkist-scraper/blob/master/README.md#quirks--known-bugs)

**Comment by Wea1thRS** [comment]
I have that enabled, yeah. It seems to work with some files and not others. It seems very inconsistent as the HTML files and such work just fine, just the m4a's sometimes have issues. But yeah. After I took it out of the string of subfolders I had, it seems to be working now much better.

**Comment by leoncvlt** [comment]
Hey, sorry about those issues, the causes are kinda hard to pin down. Might it be that you have an outdated `ffmpeg` distribution? As far as I remember the long paths incompatibility was an issue for them too, but it eventually got fixed.

**Comment by rocketinventor** [comment]
@Wea1thRS By "_out of the string of subfolders_" you mean that you changed the path that main.py was running from? I did that, too, by using linked folders, but even so, I found some titles too long (without changing the windows settings).

Have you checked your version of FFmpeg to verify that that wasn't the source of the issue? - If it is up to date, then more digging in will need to be done in order to track down this issue.
